dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/UploadValidation.java

```diff
@@ -43,7 +43,7 @@
 
         for (String key : uploadConfigurationService.getMap().keySet()) {
             if (getName().equals(key)) {
-                UploadConfiguration uploadConfig = uploadConfigurationService.getMap().get(key);
+                UploadConfiguration uploadConfig = uploadConfigurationService.getMap().get(config.getId());
```

Comment:
@benbosman : While this is a one line change, it looks slightly odd to me, as the previous line is an `if` statement looking for the "key" to match.

Does this method need further refactoring?  It almost seems like previously we had to loop through the keys in the `uploadConfigurationService` to locate the one that *matched* with `UploadValidation.getName()`.   Now, it looks like the loop has no purpose, as we're just calling `get(config.getId()` to grab the UploadConfiguration that we want directly.

Am I misunderstanding this tiny change?  It almost seems like we can remove this `for` loop entirely.